python/setup.py

```diff
@@ -232,11 +233,12 @@ def cpp_flag(compiler):
 
 library_dirs = [NGRAPH_CPP_LIBRARY_DIR]
 
-libraries = [NGRAPH_CPP_LIBRARY_NAME, ONNX_IMPORTER_CPP_LIBRARY_NAME]
+libraries = [NGRAPH_CPP_LIBRARY_NAME]
 
 extra_compile_args = []
 if NGRAPH_ONNX_IMPORT_ENABLE in ["TRUE", "ON", True]:
```

Comment:
It is copied from this line. Will check if we still need a 3-way check. Seems like the code is from the days when bool type was not normalized.